doc/wg/network/README.md

```diff
@@ -0,0 +1,43 @@
+Network Working Group (NWG)
+=================================
+
+- Working Group Charter
+- Adopted 7/28/2023
+
+## Goals
+
+The goals of the Network Tock Working Group (NWG) are to:
+
+- design a set of interfaces (traits) for networking
+- design the user land interfaces for networking
+- define how buffer management should be implemented
+
+## Members
+
+- Alexandru Radovici, OxidOS Automotive
+- Brandon Ghena, Northwestern University
+- Leon Schuermann, Princeton University
+- Tyler Potyondy, UCSD
```

Comment:
@tyler-potyondy I wanted to confirm that you're interested in being part of the working group (you can always leave in a couple months if it no longer makes sense to join calls).